lefthook.yml

```diff
@@ -0,0 +1,21 @@
+# EXAMPLE USAGE
+# Refer for explanation to following link:
+# https://github.com/evilmartians/lefthook/blob/master/docs/full_guide.md
+
+pre-push:
+  commands:
+    gems-audit:
+      tags: backend security
+      run: bundle audit
+
+pre-commit:
```

Comment:
please run all the linter from the list:
Pre-push
- bundle audit
- bundler leak
- brakeman

Pre-commit:
- rubocop